src/lib/utils.ts

```diff
@@ -19,7 +19,7 @@ import { getArgvs } from './options';
 axiosRetry(axios, {
   retries: 3,
```

Comment:
Let's also up the number of retries to 5?

Reply:
Increased!